LongWarOfTheChosen/Localization/BetterSecondWaveSupport.int

```diff
@@ -1,3 +1,11 @@
+; LWOTC Needs Translation
 [EnableChosen SecondWaveOptionObject]
-Tooltip="Enables the Chosen, who will activate after you complete the first network tower mission"
+Tooltip="Enables the Chosen, who will activate after you complete the first network tower mission."
 Description="The Chosen: Enables the Chosen"
+; End Translation
+
+; LWOTC Needs Translation
+[AllowSameRankAbilities SecondWaveOptionObject]
+Tooltip="Allows XCOM soldiers to purchase multiple perks of the same rank with AP"
+Description="Allow Same Rank Abilities: XCOM soldiers can purchase multiple perks of the same rank with AP once the Training Center has been built."
```

Comment:
I would clarify that this applies to _class_ abilities/skills/perks. Not quite sure which of those words to standardise on.

Reply:
Good catch, I think abilities would be best here.